Instructions for Examples lead to java.lang.ClassNotFoundException: io.kubernetes.client.examples.xxxExample

**Describe the bug**
Instructions for Examples lead to java.lang.ClassNotFoundException: io.kubernetes.client.examples.xxxExample

The content of the Readme files
```
examples/examples-release-10
examples/examples-release-11
examples/examples-release-12
examples/examples-release-13
```
have all the same misleading content:
```
export REPO_ROOT=/path/to/client-java/repo

cd ${REPO_ROOT}/kubernetes
mvn install

cd ${REPO_ROOT}/examples
mvn package
mvn exec:java -Dexec.mainClass="io.kubernetes.client.examples.Example"
```

If you follow those instructions, then the `mvn exec` command will give you an error 
```
java.lang.ClassNotFoundException: io.kubernetes.client.examples.Example
```
Then I have tried to do the following:
```
cd examples-release-13
mvn exec:java -Dexec.mainClass="io.kubernetes.client.examples.Example"
```
Here, we get another error message:
```
[ERROR] Failed to execute goal on project client-java-examples-release-13: Could not resolve dependencies for project io.kubernetes:client-java-examples-release-13:jar:12.0.1-SNAPSHOT: Failed to collect dependencies at io.kubernetes:client-java-api:jar:12.0.1-SNAPSHOT: Failed to read artifact descriptor for io.kubernetes:client-java-api:jar:12.0.1-SNAPSHOT: Could not find artifact io.kubernetes:client-java-parent:pom:12.0.1-SNAPSHOT -> [Help 1]
```
Newbees have a hard time with those examples, I can tell. I was interested in the InClusterClientExample, and I found that none of the versions 11, 12 (12 this is the version the Readme of the main project points to) and 13 work. Only version 10 seems to work:
Version 13:
```
cd ${REPO_ROOT}/examples/examples-release-13
mvn exec:java -Dexec.mainClass="io.kubernetes.client.examples.InClusterClientExample"
...
[ERROR] Failed to execute goal on project client-java-examples-release-13: Could not resolve dependencies for project io.kubernetes:client-java-examples-release-13:jar:12.0.1-SNAPSHOT: Failed to collect dependencies at io.kubernetes:client-java-api:jar:12.0.1-SNAPSHOT: Failed to read artifact descriptor for io.kubernetes:client-java-api:jar:12.0.1-SNAPSHOT: Could not find artifact io.kubernetes:client-java-parent:pom:12.0.1-SNAPSHOT -> [Help 1]
```
Version 12:
```
cd ${REPO_ROOT}/examples/examples-release-12
mvn exec:java -Dexec.mainClass="io.kubernetes.client.examples.InClusterClientExample"
...
[ERROR] Failed to execute goal on project client-java-examples-release-12: Could not resolve dependencies for project io.kubernetes:client-java-examples-release-12:jar:12.0.1-SNAPSHOT: The following artifacts could not be resolved: io.kubernetes:client-java-cert-manager-models:jar:12.0.1-SNAPSHOT, io.kubernetes:client-java-prometheus-operator-models:jar:12.0.1-SNAPSHOT: Could not find artifact io.kubernetes:client-java-cert-manager-models:jar:12.0.1-SNAPSHOT -> [Help 1]
```
Version 11 and Version 10 seem to work. They only have a problem with the permissions:
```
cd ${REPO_ROOT}/examples/examples-release-12
mvn exec:java -Dexec.mainClass="io.kubernetes.client.examples.InClusterClientExample"
...
io.kubernetes.client.openapi.ApiException: Forbidden
```
However, IMHO, to make it easier for newbees, the example should describe how to prepare the environment including the service account and rbac, so the example works.

** Client Version **
master (99bab18)

** Kubernetes Version **
`1.14.1`

** Java Version **
Java 11.0.9.1

**To Reproduce**
See above

**Expected behavior**
When following the Example Readmes, we should receive a success.

**KubeConfig**
Not applicable

**Server (please complete the following information):**
 - OS: Linux CentOS 7
 - native

**Additional context**
Add any other context about the problem here.

Hi @brendandburns, thanks a lot for the quick answer.

I have tried a `mvn install` on project root, but I get:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project client-java-api: Fatal error compiling: io.sundr.SundrException: No getter found for property: instance on class: io.kubernetes.client.openapi.models.V1ClusterRoleBindingListFluentImpl -> [Help 1]
```
I have tried on master on my previous commit 99bab187 as well as the latest commit bfcaea2c with the same result.

My mvn version and java version is:
```
$ mvn -v
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: /opt/apache-maven-3.6.3
Java version: 11.0.9.1, vendor: Red Hat, Inc., runtime: /usr/lib/jvm/java-11-openjdk-11.0.9.11-2.el7_9.x86_64
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "3.10.0-957.5.1.el7.x86_64", arch: "amd64", family: "unix"
```

--- a/examples/examples-release-11/README.md
+++ b/examples/examples-release-11/README.md
@@ -3,11 +3,11 @@
 ```sh
 export REPO_ROOT=/path/to/client-java/repo
 
-cd ${REPO_ROOT}/kubernetes
+cd ${REPO_ROOT}/
 mvn install
 
-cd ${REPO_ROOT}/examples
-mvn package
+cd ${REPO_ROOT}/examples/examples-10
+mvn compile
 mvn exec:java -Dexec.mainClass="io.kubernetes.client.examples.Example"
 ```
 

--- a/examples/examples-release-12/README.md
+++ b/examples/examples-release-12/README.md
@@ -3,11 +3,11 @@
 ```sh
 export REPO_ROOT=/path/to/client-java/repo
 
-cd ${REPO_ROOT}/kubernetes
+cd ${REPO_ROOT}/
 mvn install
 
-cd ${REPO_ROOT}/examples
-mvn package
+cd ${REPO_ROOT}/examples/examples-12
+mvn compile
 mvn exec:java -Dexec.mainClass="io.kubernetes.client.examples.Example"
 ```
 

--- a/examples/examples-release-12/pom.xml
+++ b/examples/examples-release-12/pom.xml
@@ -12,8 +12,8 @@
 	<name>client-java-examples-release-12</name>
 
 	<properties>
-		<kubernetes.prometheus.version>12.0.1-SNAPSHOT</kubernetes.prometheus.version>
-		<kubernetes.cert-manager.version>12.0.1-SNAPSHOT</kubernetes.cert-manager.version>
+		<kubernetes.prometheus.version>10.0.1</kubernetes.prometheus.version>
+		<kubernetes.cert-manager.version>10.0.1</kubernetes.cert-manager.version>
 	</properties>
 
 	<dependencies>

--- a/examples/examples-release-13/README.md
+++ b/examples/examples-release-13/README.md
@@ -3,11 +3,11 @@
 ```sh
 export REPO_ROOT=/path/to/client-java/repo
 
-cd ${REPO_ROOT}/kubernetes
+cd ${REPO_ROOT}/
 mvn install
 
-cd ${REPO_ROOT}/examples
-mvn package
+cd ${REPO_ROOT}/examples/examples-13
+mvn compile
 mvn exec:java -Dexec.mainClass="io.kubernetes.client.examples.Example"
 ```
 

